.circleci/config.yml

```diff
@@ -147,21 +150,21 @@ workflows:
       - doc-min-dependencies:
           requires:
             - lint
-      - pypy3:
-          filters:
-            branches:
-              only:
-                - 0.20.X
+      - pypy3
+      #    filters:
+      #      branches:
+      #        only:
+      #          - 0.20.X
       - deploy:
           requires:
             - doc
-  pypy:
-    triggers:
-      - schedule:
-          cron: "0 0 * * *"
-          filters:
-            branches:
-              only:
-                - main
-    jobs:
-      - pypy3
+  # pypy:
+  #  triggers:
+  #    - schedule:
+  #        cron: "0 0 * * *"
+  #        filters:
+  #          branches:
+  #            only:
+  #              - main
+  #  jobs:
+  #    - pypy3
```

Comment:
These need to be reverted before we can merge.

Reply:
Reverted ;)